implementation/contracts/system/TBTCDepositToken.sol

```diff
@@ -53,5 +56,7 @@ contract TBTCDepositToken is ERC721Metadata, DepositFactoryAuthority {
         spender.receiveApproval(msg.sender, _tdtId, address(this), _extraData);
     }
 }
+
+/// @notice Interface of recipient contract for approveAndCall pattern. This is implemented in FundingScript.sol
```

Comment:
I wouldn't explicitly tie this to `FundingScript.sol`. Maybe the thing to say here is “for an example, see the `FundingScript` contract”?